Formula/languagetool.rb

```diff
@@ -43,6 +43,36 @@ def install
     EOS
   end
 
+  plist_options manual: "#{HOMEBREW_PREFIX}/bin/languagetool-server --port 8081 --allow-origin"
+
+  def plist
```

Comment:
Please make this a `service` block instead of a `plist`.

Reply:
Done! Can you update the [documentation](https://docs.brew.sh/Formula-Cookbook#launchd-plist-files) sometime to point towards the preferred format?

Reply:
The `service` block API is still a work in progress, so we'll want to wait until it's closer to stable when we add it to the documentation.

Reply:
The `service` block API is still a work in progress, so we'll want to wait until it's closer to stable when we add it to the documentation.